PROJECT.md

```diff
@@ -28,8 +28,9 @@ include a [Code Header License block](#code-header-license-block).
 ```markdown
 ## License & Copyright
 
-Copyright (C) <year> Data Together.
-This program is free software; you can redistribute it and/or modify it under
+Copyright (C) <year> Data Together
```

Comment:
I want to make the case for the trailing period one more time and then will accept whatever you want to choose here.

I submit to you that the text the FSF has in their templates was never meant to lack a period at the end of the Copyright line. I believe they left it out simply because they have a placeholder at the end of the Copyright line. As evidence for this, I offer the following examples of archetypical GNU software by the FSF:

Random file from GNU make: https://github.com/mirror/make/blob/master/main.c
Random file from gcc: https://github.com/gcc-mirror/gcc/blob/master/libtool.m4
Random file from ls: https://github.com/coreutils/coreutils/blob/master/src/chmod.c

All of them have a period at the end of that line. So either they're not following their own guidelines in their own software, or it is acceptable to put a period at the end of the Copyright line.